src/CMakeLists.txt

```diff
@@ -182,6 +182,7 @@ set( MIOpen_Source
     conv/invokers/gen_x_w_y_pad.cpp
     conv/invokers/ocl_wrw_rdc.cpp
     conv/invokers/impl_gemm.cpp
+    conv/invokers/impl_gemm_dynamic.cpp
```

Comment:
Hi @atamazov I re-arrange the code and add a new invoker dedicated for dynamic igemm. hence I remove the file src/implicitgemm_dynamic.cpp